services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl

```diff
@@ -17,6 +19,8 @@ workflow arrays {
     Array[Int]? output_optional_array = []
     Array[Int?] output_array_of_optionals = []
     Array[Int?]? output_optional_array_of_optionals = []
+    Array[Int]+ output_nonempty_array = [5]
```

Comment:
Can you also explain `output non-empty array`?

Reply:
I stuck an "output" on just to distinguish it from the input variables with the same names, which is helpful when looking at the JSON

Reply:
Thank you for the explanation @aednichols! :)